package.json

```diff
@@ -51,6 +51,7 @@
     "flow-bin": "^0.245.2",
     "hermes-eslint": "^0.23.1",
     "jest": "^30.0.0-alpha.6",
+    "lightningcss": "^1.27.0",
```

Comment:
This should be added as dependency to the `rollup-plugin` and not the monorepo itself.